client/extension_link_followers.gen.go

```diff
@@ -2,7 +2,7 @@
  * Copyright (C) 2020-2025 Arm Limited or its affiliates and Contributors. All rights reserved.
  * SPDX-License-Identifier: Apache-2.0
  */
-
+// nolint
```

Comment:
Could you make it a bit granular i.e. explicitly discard the linter errors we know about
```suggestion
// nolint;stylecheck,gocritic
```